en/docs/administration/production-deployment-guideline.md

```diff
@@ -0,0 +1,72 @@
+# Production Deployment guideline
+
+The requirements for deploying WSO2 products can vary based on the deployment scenario and pattern. The recommendations in this topic are intended for general production use.
+
+WSO2 APK can be configured through `values.yaml` file.  Please refer to [Customize Configurations](../setup/Customize-Configurations.md) for information on how to use a customized values file for APK deployment. When deploying WSO2 APK in a production environment, we strongly recommend following these guidelines.
+
+
+## Change the hostnames and vhosts
+
+By default, APK uses wso2.com for its hostnames and vhosts for the gateway. You need to change these values to your own domain, which you plan to use for production. The following values.yaml values should be modified:
+
+- wso2.apk.listener.hostname
+- wso2.apk.dp.gateway.listener.hostname
+- wso2.apk.dp.configdeployer.vhosts
+
+For further clarification on the keys, please refer to the description and default values [here](https://github.com/wso2/apk/blob/main/helm-charts/README.md)
+
+## Change the gateway certificates
```

Comment:
Heading should be change certificates. Then under that topic we should discuss changing all the certs and it not just gateway components.

Reply:
Under these there are two options. With cert manager and without cert manager. We need to cover both aspects.